docs/index.rst

```diff
@@ -1487,11 +1505,14 @@ Setup |ccloud|
 
       source delta_configs/env.delta
 
+.. _cp-demo-telemetry-reporter:
 
 Telemetry Reporter
 ------------------
 
-#. Create a new ``Cloud`` API key and secret to authenticate with |ccloud|. These credentials will be used by the :ref:`telemetry_reporter` and used by the Metrics API, which can be used for hosted on-prem clusters as well as |ccloud| clusters.
+Enable :ref:`telemetry_reporter` on the on-prem cluster, and configure it to send metrics to the |ccloud| instance created above..
+
+#. Create a new ``Cloud`` API key and secret to authenticate with |ccloud|. These credentials will be used by the Telemetry Reporter and used by the Metrics API, which can be used for hosted on-prem clusters as well as |ccloud| clusters.
```

Comment:
```suggestion
#. Create a new ``Cloud`` API key and secret to authenticate with |ccloud|. These credentials are used by the Telemetry Reporter and the Metrics API, which can be used for hosted on-prem clusters as well as |ccloud| clusters.
```

Reply:
suggest: `..., which can be used for hosted` make it clear what `which` means here, does it mean the Metrics API or does it mean the API key and secret. Also, you could make this two sentences to be clearer. Put a period after Metrics API and starte the next sentence with the subject.

Reply:
Got it, I reworked this sentence to simplify it